homeassistant/components/itach/remote.py

```diff
@@ -106,19 +106,22 @@ def is_on(self):
     def turn_on(self, **kwargs):
         """Turn the device on."""
         self._power = True
-        self.itachip2ir.send(self._name, "ON", 1)
```

Comment:
I feel this could be reverted as i belive send_command is the only one that defines num_repeats support.

Reply:
The power on and power off IR commands are still required to be repeated for some devices (Sony TVs for example).  

The only other way I could think to get around this would be to modify the raw hex command definitions to repeat the required number of times, but that would prevent using the readily available IR codes.

Reply:
thats a bit odd to require 3 repeats but ive seen odder things i guess. Ill mark these issues as resolved.